transportable-udfs-examples/transportable-udfs-example-udfs/build.gradle

```diff
@@ -14,10 +14,12 @@ dependencies {
 }
 
 // If the license plugin is applied, disable license checks for the autogenerated source sets
-plugins.withId('com.github.hierynomus.license') {
-  licenseHive.enabled = false
-  licensePresto.enabled = false
-  licenseSpark.enabled = false
+tasks.whenTaskAdded { task ->
+  if (task.name.startsWith("licensePresto")
+      || task.name.startsWith("licenseHive")
+      || task.name.startsWith("licenseSpark")) {
+    task.enabled = false
+  }
 }
```

Comment:
I guess this change is no longer needed?

Reply:
removed